### PR TITLE
IGVF-1757-rename-barcode-seq

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/tests/test_audit_measurement_set.py [hunk 1]
```diff
@@ -762,3 +762,51 @@ def test_audit_missing_auxiliary_set(
...
+        }
+    )
+    testapp.patch_json(
+        measurement_set['@id'],
+        {
+            'auxiliary_sets': [base_auxiliary_set['@id']]
+        }
+    )
+    res = testapp.get(measurement_set['@id'] + '@@audit')
+    print(res.json)
```
Comment (jenjou) [inline src/igvfd/tests/test_audit_measurement_set.py:797]: Extra print here

File: src/igvfd/schemas/changelogs/auxiliary_set.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`auxiliary_set.json`*
 
+### Schema version 8
+
+* Rename `file_set_type` enum `circularized barcode detection` to `circularized RNA barcode detection`.
+* Rename `file_set_type` enum `quantification DNA barcode sequencing` to `circularized RNA barcode detection`.
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/auxiliary_set.md:6]: Looks like a copy paste error

File: src/igvfd/audit/measurement_set.py [hunk 3]
```diff
@@ -270,3 +270,29 @@ def audit_missing_auxiliary_sets(value, system):
...
+        }
+    ]
+    '''
+    description = get_audit_description(audit_missing_auxiliary_set_MPRA)
+    assay_term = value.get('assay_term')
+    assay_object = system.get('request').embed(assay_term, '@@object?skip_calculated=true')
+    assay_term_name = assay_object.get('term_name')
+    if assay_term_name == 'massively parallel reporter assay':
+        auxiliary_sets = [system.get('request').embed(auxiliary_set, '@@object?skip_calculated=true')
+                          for auxiliary_set in value.get('auxiliary_sets', '')]
```
Comment (jenjou) [inline src/igvfd/audit/measurement_set.py:292]: In `value.get('auxiliary_sets', '')` the `''` should be a `[]` (return empty array if the get() doesn't get anything).

File: src/igvfd/audit/measurement_set.py [hunk 4]
```diff
@@ -270,3 +270,29 @@ def audit_missing_auxiliary_sets(value, system):
...
+        if not (auxiliary_sets) or not ([auxiliary_set for auxiliary_set in auxiliary_sets if auxiliary_set.get('file_set_type') == 'quantification DNA barcode sequencing']):
```
Comment (jenjou) [inline src/igvfd/audit/measurement_set.py:293]: Not really necessary as it is required, but would be good practice to add  a `''` in `auxiliary_set.get('file_set_type', '')`

**Review by jenjou — CHANGES_REQUESTED** [review]
Generally looks ok, just some minor issues to fix

File: src/igvfd/tests/test_audit_measurement_set.py (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/tests/test_audit_measurement_set.py:797]: fixed

File: src/igvfd/schemas/changelogs/auxiliary_set.md (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/auxiliary_set.md:6]: fixed

File: src/igvfd/audit/measurement_set.py (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/audit/measurement_set.py:292]: fixed

File: src/igvfd/audit/measurement_set.py (hunk 4 above)
Comment (ian-whaling) [reply src/igvfd/audit/measurement_set.py:293]: fixed

**Review by jenjou — APPROVED** [review]

**Pull request merged**